.github/workflows/post-review-ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
   # JikesRVM
   jikesrvm-binding-test:
     runs-on: ubuntu-18.04
-    timeout-minutes: 60
+    timeout-minutes: 90
```

Comment:
I will revert this to 60 after the CI is finished and I removed the benchmarking/testing label